### PR TITLE
Feature/add abstract property and abstract entities

### PR DESCRIPTION
**Pull request by michelu89**
Dieser Pull Request beinhaltet die Erstellung von Abstract Properties und Abstract Entities.

### CODE REVIEW

File: core/apps/ame-e2e/src/fixtures/abstract-entity.txt [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+    bamm:operations ();
+    bamm:events ().
+:property1 a bamm:Property;
+    bamm:characteristic :Characteristic1.
+:Characteristic1 a bamm:Characteristic;
+    bamm:dataType :Entity1.
+:Entity1 a bamm:Entity;
+    bamm:properties (:property2 :property3);
+    bamm:extends :AbstractEntity1.
+:property2 a bamm:Property;
```
Comment (georgschmidtdumont) [inline core/apps/ame-e2e/src/fixtures/abstract-entity.txt:21]: I am not sure what you want to use this Aspect model for, but the definition for `property2` and `property3` is not valid. The characteristic is missing for these properties.

File: core/apps/ame-e2e/src/fixtures/time-series-entity.txt [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+:AspectDefault a bamm:Aspect;
+    bamm:properties (:property1);
+    bamm:operations ();
+    bamm:events ().
+:property1 a bamm:Property;
+    bamm:characteristic :Characteristic1.
+:Characteristic1 a bamm:Characteristic;
+    bamm:dataType :Entity1.
+:Entity1 a bamm:Entity;
+    bamm:properties ([bamm:extends bamm-e:value]);
```
Comment (georgschmidtdumont) [inline core/apps/ame-e2e/src/fixtures/time-series-entity.txt:19]: Same here. I am not sure what you want to use this Aspect model for, but the definition for this property is not valid. The characteristic is missing.

Comment (michelu89) [reply core/apps/ame-e2e/src/fixtures/time-series-entity.txt:19]: In this case we test exactly this case, that we can load it but it is invalid.

File: core/apps/ame-e2e/src/fixtures/abstract-entity.txt (hunk 1 above)
Comment (michelu89) [reply core/apps/ame-e2e/src/fixtures/abstract-entity.txt:21]: In this case we test exactly this case, that we can load it but it is invalid.

File: core/apps/ame-e2e/src/fixtures/abstract-entity.txt [hunk 3]
```diff
@@ -0,0 +1,39 @@
+@prefix bamm: <urn:bamm:io.openmanufacturing:meta-model:2.0.0#>.
```
Comment (georgschmidtdumont) [inline core/apps/ame-e2e/src/fixtures/abstract-entity.txt:1]: Please add the copyright header.

File: core/apps/ame-e2e/src/fixtures/time-series-entity.txt [hunk 4]
```diff
@@ -0,0 +1,23 @@
+@prefix bamm: <urn:bamm:io.openmanufacturing:meta-model:2.0.0#>.
```
Comment (georgschmidtdumont) [inline core/apps/ame-e2e/src/fixtures/time-series-entity.txt:1]: Please add the copyright header.

**Review by georgschmidtdumont — CHANGES_REQUESTED** [review]

**Review by georgschmidtdumont — APPROVED** [review]

**Pull request merged**